src/main.cpp

```diff
@@ -0,0 +1,27 @@
+al#include <Arduino.h>
+#include "mcp23017.h"
+
+// TODO: declare mcp23017 object
+Mcp23017 mcp23017(0x20);
+
+
+void setup() {
+  // TODO: initialize i2c and mcp23017 object
+  Wire.begin();
+  uint8_t directions[8] = {1, 1, 1, 1, 1, 1, 0};
```

Comment:
Check the data sheet to verify what 0 or 1 means for the direction (read or write)